gqcnn/model/tf/network_tf.py

```diff
@@ -552,6 +552,19 @@ def initialize_network(self,
             if add_sigmoid:
                 self.add_sigmoid_to_output()
 
+            # Freeze graph. Make it 'True' to freeze this graph
```

Comment:
Would it be possible to automate this as part of `gqcnn/model/openvino/network_openvino.py`? I just don't like the user having to edit the source. See https://github.com/BerkeleyAutomation/gqcnn/blob/ncs/gqcnn/model/tf/ncs_fc_network_tf.py#L251 for an example. There when the user invokes `compile()`, which executes the compilation shell commands, the first step is to generate the protobuf.

Reply:
Yes I think the freezing codes could be put into a separate function (e.g. `freeze()`), like what you did for `ncs_fc_network_tf.py`.
An open question is where to invoke the `freeze()`. I noticed `compile()` is invoked each time when the FCGQCNN model is loaded.
While `freeze()` is necessary only for OpenVINO model conversion, which is usually done once the pre-trained model are updated. Would you mind if `freeze()` is invoked each time loading the GQCNN model?